design-documents/graph-ql/coverage/negotiableQuotes.graphqls

```diff
@@ -0,0 +1,147 @@
+type Mutation {
```

Comment:
Mutations for managing negotiable quote items are missing.

Reply:
@paliarush both of those should be supported. In the admin today merchants can remove and add products, including configurable products or those with custom options.